platform/solr/solr-dependencies/pom.xml

```diff
@@ -156,8 +166,8 @@
                             solr-solrj,
                             commons-io,
                             httpmime,
-                            noggit,
                             zookeeper,
+                            zookeeper-jute,
```

Comment:
❓ I was reading the docs and it looks like this is for Hadoop APIs? Is it a case where we need it because it's a transitive but we might not be invoking it? 

See: https://github.com/apache/zookeeper/blob/release-3.5.5/zookeeper-jute/src/main/java/org/apache/jute/package.html

Reply:
I think Hadoop is where they copied Jute from originally.  Zookeeper uses it to serialize data into znodes. The Solr upgrade did not work until adding this dependency because the Zookeeper client used by the Solr clients needs it at runtime.

https://github.com/apache/zookeeper/search?q=jute+record&unscoped_q=jute+record
https://lucene.apache.org/solr/guide/8_5/setting-up-an-external-zookeeper-ensemble.html